software/controller/test/nvparams/nvparams_test.cpp

```diff
@@ -19,12 +19,11 @@ limitations under the License.
 
 using namespace NVParams;
 
-static NVParams::Handler nv_params;
 static constexpr uint32_t kMemSize{8192};
```

Comment:
could kMemSize be a static member of the new NVparamsTest class, as it is only used there?